### DISCUSSION TITLE
Screenshots of editor

### DISCUSSION THREAD

**Discussion by icarusinthewings** [NovaoTech/solaris#24] [Announcements]
Heyo!
I have a preview of the styling for the editor that we'll be using.
I've accomplished this using ScratchAddons and it will be available as a userstyle soon.
![Screenshot from 2021-10-16 22-27-42(1)](https://user-images.githubusercontent.com/24457862/137608390-51875e95-7549-4455-997f-324a74ceadf7.png)
Here is the information if you want to test out the UI itself:
![Screenshot from 2021-10-16 22-35-25](https://user-images.githubusercontent.com/24457862/137608416-c13559b2-522a-40fa-8d00-f8404fd45972.png)
![Screenshot from 2021-10-16 22-35-16](https://user-images.githubusercontent.com/24457862/137608417-2db8c235-c805-42c0-b761-34f1c5779869.png)
The accent color theme is turbowarp with the accent color as #6860e4.
Replace the logo with `<img alt="Scratch" class="menu-bar_scratch-logo_2uReV menu-bar_clickable_1g3uo" draggable="false" src="https://polaris.velocitydev.repl.co/static/wordmark.svg">` and modify the height to be `1.3rem`
That's all
~ Velocity

**Comment by spazestudios** [comment]
yooo

**Comment by MikeDev101** [comment]
Now that's quack-a-lack'n!

**Comment by CloudGate1234** [comment]
I never used the scratch addons dark mode because it made the costume editing area too dark. Although the rest of the website is dark, the editor isn't for me. I could make a polaris userstyle for the entire website that includes my supercool "darker than black" userscript. link: https://userstyles.world/style/1257/darker-than-black-scratch-theme

**Reply by CloudGate1234** [reply]
I am also working on a "gamer" theme. sorta like opera gx here is a preview:
![Screenshot 2021-10-22 150638](https://user-images.githubusercontent.com/88938765/138511068-474a96b5-7a15-4db0-b5a2-f639f1b02d90.png)

**Reply by MikeDev101** [reply]
"High contrast but with mountain dew"

**Comment by spazestudios** [comment]
what setting is that outline on the blocks I need it

**Reply by devxan** [reply]
It's the **Customizable block colors addon** with Text Color: Colored on black background 
<img width="699" alt="image" src="https://user-images.githubusercontent.com/57809064/153516638-d0a26419-3b15-4c40-99c4-6f1cabde8df9.png">
_[I am a Scratch Addons contributor.](https://scratchaddons.com/contributors/#:~:text=devxan) I'm also really confused what this is._

**Reply by MikeDev101** [reply]
> I'm also really confused what this is.

This is a Scratch community clone that aims to be more open to ideas and rules. We're still a WIP.

**Reply by devxan** [reply]
> This is a Scratch community clone that aims to be more open to ideas and rules. We're still a WIP.

So how will people commune?

**Reply by MikeDev101** [reply]
We're still in the planning stages for that, but we have talked about using [Meower](https://github.com/meower-media-co/) for parts of the chat backend. @VelocityDesign correct me if I'm wrong.
